modules/dnn/perf/perf_net.cpp

```diff
@@ -233,20 +269,96 @@ PERF_TEST_P_(DNNTestNetwork, Inception_v2_Faster_RCNN)
         (backend == DNN_BACKEND_OPENCV && target == DNN_TARGET_OPENCL_FP16))
         throw SkipTestException("");
     processNet("dnn/faster_rcnn_inception_v2_coco_2018_01_28.pb",
-               "dnn/faster_rcnn_inception_v2_coco_2018_01_28.pbtxt", Mat(cv::Size(800, 600), CV_32FC3));
+               "dnn/faster_rcnn_inception_v2_coco_2018_01_28.pbtxt", cv::Size(800, 600));
```

Comment:
Prefer to keep code formatting from 4.x branch to avoid merge conflicts in the future:

https://github.com/opencv/opencv/blob/7757759205aa0040d9f277a06edff92e70c6634c/modules/dnn/perf/perf_net.cpp#L324-L326

Reply:
There was signature change in 5.x after Halide removal. So there is a conflict anyway, if I presume formatting.

Reply:
Origin is 4.x.
Halide removal PR change formatting here wrongly.

We should prefer original formatting (from base branch) to minimize merge conflicts in the future.

Reply:
Fixed.